### DISCUSSION TITLE
Migrate from signal-cli Docker container to HA Add-On

### DISCUSSION THREAD

**Discussion by burnsie-la** [bbernhard/signal-cli-rest-api#321] [Q&A] [answered]
I am currently running the signal-cli-rest-api in a Docker container via Docker compse, and I am looking to migrate my config to the HA Add on.  Is it possible to take my /config directory from the Docker (With its avatars and data sub-directories) and somehow get them onto the Add On container so I don't have to set up all my number registration and groups again?

**Comment by bbernhard** [comment, marked as answer]
Unfortunately I do not know much about the HA addon, but if you can find a way to bind mount the existing signal-cli config folder into the HA addon, it should work
